### ISSUE TITLE
Wonder more pretrain scripts and results

### ISSUE THREAD

**Issue by LinB203** [OpenGVLab/VideoMAEv2#18]
Wonderful work.
In paper's Table.2, you report the Top-1 accuary about the ViT-B on Something-Something v2 by using dual masking method, but there is no pretrain scripts in this repo. 
So I wonder that is the pretrain script same with VideoMAE v1? Did you pretrain on Kinetic400 by using dual masking method and how about results?

**Comment by congee524** [comment]
Yes. Just use the same script and add the two lines
```shell
    --decoder_mask_type run_cell \
    --decoder_mask_ratio 0.5 \
```

We didn't pre-train on K400 with dual masking.

**Comment by LinB203** [comment]
thx, clearly.
